addons/grenades/stringtable.xml

```diff
@@ -508,5 +508,96 @@
             <Czech>M992 Infračervená světlice</Czech>
             <Turkish>M992 Illumination IR Flare</Turkish>
         </Key>
+        <Key ID="STR_ACE_Grenades_SatchelCharge_Remote_Mag_Throwable">
+            <English>Explosive Satchel (Throwable)</English>
+            <German></German>
+            <Polish></Polish>
+            <Italian></Italian>
+            <Japanese></Japanese>
+            <Russian></Russian>
+            <Portuguese></Portuguese>
+            <Chinese></Chinese>
+            <French></French>
+            <Czech></Czech>
+            <Turkish></Turkish>
+        </Key>
+        <Key ID="STR_ACE_Grenades_SatchelCharge_Name">
+            <English>Satchel Charge</English>
+            <German></German>
+            <Polish></Polish>
+            <Italian></Italian>
+            <Japanese></Japanese>
+            <Russian></Russian>
+            <Portuguese></Portuguese>
+            <Chinese></Chinese>
+            <French></French>
+            <Czech></Czech>
+            <Turkish></Turkish>
+        </Key>
+        <Key ID="STR_ACE_Grenades_SatchelCharge_Description">
+            <English>Type: Charge&lt;br /&gt;Rounds: 1&lt;br /&gt;Used on: Things that need to die</English>
+            <German></German>
+            <Polish></Polish>
+            <Italian></Italian>
+            <Japanese></Japanese>
+            <Russian></Russian>
+            <Portuguese></Portuguese>
+            <Chinese></Chinese>
+            <French></French>
+            <Czech></Czech>
+            <Turkish></Turkish>
+        </Key>
+        <Key ID="STR_ACE_Grenades_SatchelCharge_LibText">
+            <English>An explosive satchel that is throwable. 7 second fixed fuse</English>
+            <German></German>
+            <Polish></Polish>
+            <Italian></Italian>
+            <Japanese></Japanese>
+            <Russian></Russian>
+            <Portuguese></Portuguese>
+            <Chinese></Chinese>
+            <French></French>
+            <Czech></Czech>
+            <Turkish></Turkish>
+        </Key>
+        <Key ID="STR_ACE_Grenades_DemoCharge_Remote_Mag_Throwable">
+            <English>Explosive Charge (Throwable)</English>
+            <German></German>
+            <Polish></Polish>
+            <Italian></Italian>
+            <Japanese></Japanese>
+            <Russian></Russian>
+            <Portuguese></Portuguese>
+            <Chinese></Chinese>
+            <French></French>
+            <Czech></Czech>
+            <Turkish></Turkish>
+        </Key>
+        <Key ID="STR_ACE_Grenades_DemoCharge_Name">
```

Comment:
This one could be replaced with `str_a3_cfgmagazines_democharge0`

Reply:
Ah nice, I was wondering if I could use vanilla string table entries